service/src/test/java/org/cbioportal/genome_nexus/service/internal/VariantAnnotationServiceTest.java

```diff
@@ -111,15 +132,17 @@ public void getAnnotationsByVariantString()
     public void getMutationAssessorEnrichedAnnotationByVariantString()
         throws ResourceMappingException, VariantAnnotationWebServiceException, VariantAnnotationNotFoundException,
         MutationAssessorWebServiceException, MutationAssessorNotFoundException, IsoformOverrideNotFoundException,
-        IOException
+        IOException, MyVariantInfoWebServiceException, MyVariantInfoNotFoundException
     {
         Map<String, VariantAnnotation> variantMockData = this.variantAnnotationMockData.generateData();
         Map<String, MutationAssessor> maMockData = this.mutationAssessorMockData.generateData();
         Map<String, IsoformOverride> isoformOverrideMockData = this.isoformOverrideMockData.generateData();
+        Map<String, MyVariantInfo> mviMockData = this.myVariantInfoMockData.generateData();
 
         this.mockVariantFetcherMethods(variantMockData);
         this.mockMutationAssessorServiceMethods(variantMockData, maMockData);
         this.mockIsoformOverrideServiceMethods(isoformOverrideMockData);
+        this.mockMyVariantInfoServiceMethods(variantMockData, mviMockData);
```

Comment:
This is not the right place to test this. We should add another method like `getMyVariantInfoEnrichedAnnotationByVariantString`.